modelon/impact/client/entities.py

```diff
@@ -1694,22 +1693,18 @@ def experiment_id(self):
         return self._exp_id
 
     def _get_info(self):
```

Comment:
Can we remove this method? This was never intended to be used by the user right?

Reply:
Fixed!